lib/rules/require-emit-types.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Emit definitions should be detailed
+ * @author Pig Fang
+ */
+'use strict'
+
+const utils = require('../utils')
+
+/**
+ * @typedef {import('../utils').ComponentArrayEmit} ComponentArrayEmit
+ * @typedef {import('../utils').ComponentObjectEmit} ComponentObjectEmit
+ */
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'require type definitions in emits',
+      categories: ['vue3-strongly-recommended'],
```

Comment:
Could you change the category to undefined as we will not change the configuration except for major version releases?